matrix_sdk_base/src/client.rs

```diff
@@ -898,13 +906,18 @@ impl BaseClient {
     /// * `room_id` - The unique id of the room the event belongs to.
     ///
     /// * `event` - The presence event for a specified room member.
-    pub async fn receive_ephemeral_event(&self, room_id: &RoomId, event: &NonRoomEvent) -> bool {
-        match event {
-            NonRoomEvent::IgnoredUserList(iu) => self.handle_ignored_users(iu).await,
-            NonRoomEvent::Presence(p) => self.receive_presence_event(room_id, p).await,
-            NonRoomEvent::PushRules(pr) => self.handle_push_rules(pr).await,
-            _ => false,
-        }
+    pub async fn receive_ephemeral_event(
+        &self,
+        _room_id: &RoomId,
```

Comment:
Why are we keeping the room id argument here if it's not needed? Will it be needed?

Reply:
Is there a reason for the method It doesn't affect the state at all should I remove it?

Reply:
I would think the [author](https://github.com/matrix-org/matrix-rust-sdk/commit/4dbc0aaad65c571f7356ebae15acdfe07e74c87e#diff-10291e402ffffc5697c84bf0bf259727R331) of the method will know the answer to this question. :stuck_out_tongue_winking_eye:.

We can remove it, but we'll probably will want to store typing and the receipts eventually.

Reply:
Hahaha yea more what I meant was if we should keep it for future sake. :laughing:

Reply:
I'll just leave it for now if we are going to eventually add those events might as well have all the plumbing ready.